libevmasm/Instruction.h

```diff
@@ -104,10 +104,9 @@ enum class Instruction: uint8_t
 	MSIZE,                    ///< get the size of active memory
 	GAS,                      ///< get the amount of available gas
 	JUMPDEST,                 ///< set a potential jump destination
-	MCOPY = 0x5e,             ///< copy between memory areas
-
 	TLOAD = 0x5c,             ///< load word from transient storage
 	TSTORE = 0x5d,            ///< save word to transient storage
+	MCOPY = 0x5e,             ///< copy between memory areas
```

Comment:
Values are sequential with no gap, no need to specify them individually
```suggestion
	TLOAD,                    ///< load word from transient storage
	TSTORE,                   ///< save word to transient storage
	MCOPY,                    ///< copy between memory areas
```

Reply:
Sure, but we keep some of them for better readability of the opcodes. Maybe we should have it for all to be honest, but this is my own opinion. Anyway, I think the change is good enough for now ;)